src/functions/template-tags/html.php

```diff
@@ -123,8 +123,72 @@ function tribe_disabled( $disabled, $echo = true ) {
 
 		if ( ! $echo ) {
 			return $disabled_string;
-		} else {
-			echo $disabled_string;
 		}
+
+		echo $disabled_string;
+	}
+}
+
+/**
+ * Adds rel attribute for links.
+ *
+ * @since TBD
+ *
+ * @param string|array $rel  Value(s) for rel attribute.
+ * @param boolean      $echo Whether to echo the string or return it.
+ *
+ * @return string the rel attribute
+ */
+function tribe_rel( $rel, $echo = true ) {
+	if ( empty( $rel ) ) {
+		return;
+	}
+
+	$attr = 'rel="' . esc_attr( $rel ) . '"';
+
+	if ( ! $echo ) {
+		return $attr;
 	}
+
+	echo $attr;
+}
+
+/**
+ * Adds target attribute for links.
+ * If target is '_blank' also adds appropriate rel attributes.
+ *
+ * @since TBD
+ *
+ * @param string  $target   Value for the target attribute.
+ * @param array|string $rel Value(s) for rel attribute.
+ * @param boolean $echo     Whether to echo the string or return it.
+ *
+ * @return string The rel attribute.
+ */
+function tribe_target( $target, $rel = [], $echo = true ) {
+	if ( empty( $rel ) ) {
+		return;
+	}
+
+	if ( ! is_array( $rel ) ) {
+		$rel = explode( ' ', $rel );
+	}
+
+	$attr = 'target="' . esc_attr( $target ) . '"';
+
+	if ( '_blank' === $target ) {
+		$rel[] = 'noopener';
+		$rel[] = 'nofollow';
+	}
+
+	if ( ! empty( $rel ) ) {
+		$attr .= tribe_rel( $rel, false );
```

Comment:
Here, you'd get `'target="_blank"rel="array array"'`. or whatever the a stringified array outputs to be. Maybe add each attribute string to an `$attr` array and implode at the end.